Packs/Armis/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Armis Event Collector
+
+Updated the supported events to fetch alerts, activities, and devices.
```

Comment:
Please add to the release notes a mention about changed max size.
If there are other documentation notes about this size, please update them accordingly.